app/scripts/controllers/util/error.js

```diff
@@ -8,16 +8,18 @@
  * Controller of the openshiftConsole
  */
 angular.module('openshiftConsole')
-  .controller('ErrorController', function ($scope, $window) {
+  .controller('ErrorController', function ($scope, $location, $window, $uibModal, HomePagePreference, NotificationsService) {
     var params = URI(window.location.href).query(true);
     var error = params.error;
 
     switch(error) {
       case 'access_denied':
         $scope.errorMessage = "Access denied";
+        checkHomePagePreference(params.project_name);
```

Comment:
It would be nice if we can catch this somewhere before we redirect to the error page.

I think this will actually incorrectly show the message if you go to

```
/projects/my-project/pod/missing-pod
```

even if my-project exists